### ISSUE TITLE
navigator.clipboard.readText throws in Firefox

### ISSUE THREAD

**Issue by ralake** [glideapps/glide-data-grid#349]
If you paste data into a text cell in Firefox, the following error is thrown...

```
Uncaught (in promise) TypeError: navigator.clipboard.readText is not a function
```

Following the steps in this post stops the error throwing - https://stackoverflow.com/questions/67440036/navigator-clipboard-readtext-is-not-working-in-js

Is this a known issue for this package? I guess for now I'll have to find a way to disable pasting data for Firefox users...

**Comment by jassmith** [comment]
I will get a hotfix out.

**Comment by jassmith** [comment]
What verison of Glide Data Grid are you using?

**Comment by ralake** [comment]
> What verison of Glide Data Grid are you using?

4.0.3

**Comment by jassmith** [comment]
I'm not able to right this second, could you test if I fixed this in 4.1.0-beta1

**Comment by ralake** [comment]
> I'm not able to right this second, could you test if I fixed this in 4.1.0-beta1

I will have a look first thing Monday. Thanks!

**Comment by jassmith** [comment]
My testing says this is fixed in 4.1.0, please re-open (or open a new bug since I think you cant reopen) if this is not the case.
